server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ContestCounter.java

```diff
@@ -126,7 +125,12 @@ public static Set<Integer> pairwiseMargins(final Set<String> winners,
                               .map(cr -> cr.county())
                               .collect(Collectors.toSet()));
 
-    final Long ballotCount = BallotManifestInfoQueries.totalBallots(contestResult.countyIDs());
+    Long ballotCount;
+    if(useManifests) {
```

Comment:
Would it be worthwhile writing a log message to indicate that ballot manifests were/were not used in contest counting?